src/d3a/d3a_core/util.py

```diff
@@ -527,6 +537,25 @@ def convert_area_throughput_kVA_to_kWh(transfer_capacity_kWA, slot_length):
         if transfer_capacity_kWA is not None else 0.
 
 
+def find_timestamp_of_same_weekday_and_time(indict, time_slot, ignore_not_found=False):
+    if IS_CANARY_NETWORK:
+        start_time = list(indict.keys())[0]
+        key = datetime(year=start_time.year, month=start_time.month, day=start_time.day,
```

Comment:
A bit more verbose variable name would be easier to understand its purpose.

Reply:
agreed